testsuite/tools/expect_test.ml

```diff
@@ -336,7 +336,8 @@ let main fname =
   end;
   Compmisc.init_path ();
   Toploop.initialize_toplevel_env ();
-  Sys.interactive := false;
+  (* We are in interactive mode and should record directive error on stdout *)
+  Sys.interactive := true;
```

Comment:
`expect_test` is a non-interactive tool, so it makes sense to me that `Sys.interactive` is set to `false`.

I wrote this code a long time ago and I can't remember if this line was important. However, getting nice warnings and error messages that don't change all the time was the main difficulty of setting up expectation tests using the toplevel. So if setting `Sys.interactive` to `false` was important and setting it to `true` would break something, I would expect the breakage to be apparent in the testsuite.

> Currently, this change also makes expect_test looks at the file system rather than to the cache (see utlils/load_path.ml) in case of file system access. However, this behavior was added in 4.13 .

This might make the testsuite slower then. Seems worth benching.

Reply:
I suppose the impact would be larger on Windows, but I don´t observe any AppVeyor timing differences [on this PR](https://ci.appveyor.com/project/avsm/ocaml/builds/39905824) and [on another PR](https://ci.appveyor.com/project/avsm/ocaml/builds/39896593), so it looks like there is no observable performance impact.

(I think this makes sense given that many the eexpect_test in our testsuite don't rely much on external modules.)

Reply:
Yes, since the few runs of the testsuite are still within the usual range, the performance impact seems small enough.